auction-app-backend/src/main/resources/db/migrations/V9__add_payment_info_table.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE payment_info (
```

Comment:
these queries are executed independently because i didnt wrap them in a transaction block, so even if one query fails, other queries changes to the db will remain